decidim-admin/README.md

```diff
@@ -42,6 +42,19 @@ There are some pages that exist by default and cannot be deleted since there
 are links to them inside the Decidim framework, see `Decidim::StaticPage` for
 the default list.
 
+### Pager Configuration
+
+The number of results shown per page and per page range can be configured in the app `decidim.rb` initializer as follows:
+
+```ruby
+Decidim::Admin.configure do |config|
+  config.range_per_page = [15, 50, 100]
+end
+```
+
+* `range_per_page.first` sets the default value
+* `range_per_page.last` sets the max_per_page
```

Comment:
As the `per_page` is passed with a GET it could be changed manualy to a very high number, setting the `max_per_page` prevents this,  as the default value is `nil` https://github.com/kaminari/kaminari#general-configuration-options

Reply:
https://github.com/decidim/decidim/blob/b551c1cec8af3d36f5ef3c3d93ef88d0cdbc3920/decidim-admin/lib/decidim/admin.rb#L27-L30